config/templates/etc/filebeat/filebeat.yml.j2

```diff
@@ -71,6 +71,13 @@ filebeat.inputs:
   # Note: After is the equivalent to previous and before is the equivalent to to next in Logstash
   #multiline.match: after
 
+{% if filebeat_close_timeout is defined %}
```

Comment:
Can this follow the convention of being inside the other `beats_config` options?  i.e.
```
{% if beats_config.filebeat.close_timeout is defined %}
  close_timeout: {{ beats_config.filebeat.close_timeout }}
{% endif %}
```

Reply:
Yes, sure. I'll get this fixed.